packages/datatrak-web/src/features/Survey/SurveyContext/SurveyContext.tsx

```diff
@@ -22,6 +22,8 @@ const defaultContext = {
   numberOfScreens: 0,
   screenNumber: 1,
   screenHeader: '',
+  surveyCode: '',
```

Comment:
The surveyCode will always be in the url right? Will it cause potential bugs to set it somewhere else as well?

Reply:
Yes you are right and this was just left over from a previous implementation and it's not needed anymore anyway